### ISSUE TITLE
Only Playlists Detected by Clients

### ISSUE THREAD

**Issue by Prn-Ice** [jellyfin/mopidy-jellyfin#87]
Hi, on my clients, ncmpcpp, Iris only the playlists are detected in the media libraries. Albums, tracks, etc are not and can only be found by browsing the file view, is this default behavior.

**Comment by mcarlton00** [comment]
What is your music library(libraries?) named?  By default it just looks for "Music" when populating the library, however if it's named something else you'll have to define that in the config file.

Example config file line:
```
libraries = Library1, Library2 (Optional: will default to "Music" if left undefined)
```

Iris is also a weird case that doesn't use the library in a super straightforward way, and you'll need to go into it's settings page and change the artists URI to `jellyfin:artists`

**Comment by Prn-Ice** [comment]
I have two libraries, Music and Music Videos
This is my jellyfin conf section
```sh
[jellyfin]
hostname = http://127.0.0.1:8096
username = havoc
password = 4rAms
libraries = Music
watched_status = True
```
I'll try your suggestion on iris artists

**Comment by Prn-Ice** [comment]
After, adding the jellyfin:artists uri, artists load fine, thanks.

**Comment by mcarlton00** [comment]
Music videos currently won't work, and I don't have much of a plan to support them.  It's just a lot more overhead involved to strip the audio out.

Is ncmpcpp working properly now as well?  I can't think of any situation where it wouldn't be but Iris would.

**Comment by mcarlton00** [comment]
Closing due to inactivity

**Comment by Bitals** [comment]
I currently have the same problem with M.A.L.P.
Curiously, Iris worked fine out of the box, and I currently use it, but having to open a browser/web app (the same thing) on Android is less than ideal. Especially with how resource-hungry they all are now.

Could this be reopened, or should I create a new issue?

P.S. also only album arts are loaded even on Iris, no artist covers and no whatever image it is looking for a song at the screenshot below. Looks weird.

<img width="1592" height="1107" alt="Image" src="https://github.com/user-attachments/assets/3f3fe6b3-065a-4cec-8900-13fc1cc0e73a" />

P.P.S. After hours of dealing with the mess that is android clients for mopidy, I discovered that a regular Jellyfin app can just cast to mopidy???? No need for any mopidy client at all. Even though the Jellyfin wiki clearly states that DLNA only works locally, and my Jellyfin server is in another country (with a site-to-site VPN though), and my client devices running the app (laptop and phone) also are in a different VLAN from my local mopidy server. Which one of the factors helped this to just freaking work -- I have no idea, but it does, so I am ready to leave it be and never think about it again.
Sharing for history and others who have this issue I guess.

P.P.P.S I discovered that the current playlist (Now Playing) can desync when you remove tracks from the Jellyfin side. The removal progress is very choppy by itself, and when it finally looks like it propagated to both sides, I ended up with a duplicate track on the Jellyfin side, and all my currently playing tracks are offset by one lol. Meanin Jellyfin reports the previous track as currently playing, while in reality it's the next one in the queue. Seems like just an index mismatch, but I didn't find an easy way to reset this yet without creating a whole different queue. I am unable to simply remove the duplicate track. Well, I guess that is serviceable overall.
